edx_sga/tests/integration_tests.py

```diff
@@ -139,52 +140,53 @@ def make_student(self, block, name, make_state=True, **state):
                 answer[key] = state.pop(key)
         score = state.pop('score', None)
 
-        user = User(username=name)
-        user.save()
-        profile = UserProfile(user=user, name=name)
-        profile.save()
-        if make_state:
-            module = StudentModule(
-                module_state_key=block.location,
-                student=user,
+        with transaction.atomic():
```

Comment:
What's the purpose of this? All tests already run within a transaction

Reply:
making tow students in loop without this throws error that block should atomic